crmsh/utils.py

```diff
@@ -26,6 +26,7 @@
 from contextlib import contextmanager, closing
 from stat import S_ISBLK
 from lxml import etree
+from packaging import version
```

Comment:
Pacemaker is not a python package. Python packaging tools should not be used to parse it.

Reply:
This is recommended by PEP632 https://peps.python.org/pep-0632/#migration-advice

Reply:
Yeah, it is recommended to replace distutils. However, both of `distutils` and `packaging` and designed for python packages, not an unrelated project written in C.

Reply:
I don't get your point here.
We are trying to compare the `string` like "pacemaker-3.7" and "pacemaker-3.6"

Reply:
Python's [Version Specifier Specification](https://packaging.python.org/en/latest/specifications/version-specifiers/) is quite complex. `packaging` is designed to handle it. It is not the expected use case of `packaging` to parse some version string from random, non-python projects. Using it to parse the version string of pacemaker causes confusions by implying pacemaker will follow python's specification.

Reply:
Then, what's your suggestion? any good alternative?

Reply:
I think `utils.is_larger_than_min_version` is not only for parsing Pacemake's version. It's a common function